extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -276,6 +277,17 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	async addNotebookToBook(): Promise<void> {
+		const books: BookModel[] = this.books.filter(b => {
+			if (!b.isNotebook) {
+				return b;
+			}
+			return undefined;
+		});
+		const dialog = new AddNotebookDialog(this.bookTocManager, books);
+		dialog.createDialog();
```

Comment:
Not going to await on it finishing? Either update the name to `openAddNotebookToBookDialog` or make it wait until the book is actually added